global:
  ros__parameters:
    topic_prefix: '/fb'
    localization_type: "gps_and_deg" # "single_gps" or "dual_gps" or "gps_and_deg"

antenna_split:
  ros__parameters:
    name: 'split'
    #input_topic:
    gps_topic: '/gps'
    #output_topics:
    gps_main: '/gps_main'
    gps_aux: '/gps_aux'
    kallman_type: 1

antenna_fuse:
  ros__parameters:
    name: 'fuse'
    #input_topics:
    gps_main: 'fixposition/gnss1'
    gps_aux: 'fixposition/gnss2'
    angle_gpses: 90.0

gps_and_deg:
  ros__parameters:
    name: 'gps_and_deg'
    #input_topics:
    gps_corr: '/corr/gps'
    angle_deg: '/corr/heading'

gps_to_enu:
  ros__parameters:
    name: 'gps_to_enu'

odom_n_path:
  ros__parameters:
    name: 'odometry'

transform_pub:
  ros__parameters:
    name: 'transform_pub'
